ekg-json.cabal

```
@@ -24,10 +24,10 @@ library
   exposed-modules:
     System.Metrics.Json
   build-depends:
-    aeson >=0.4 && < 1.6,
-    base >= 4.6 && < 4.16,
+    aeson >= 2.0 && < 2.1,
+    base >= 4.6 && < 4.17,
```

```suggestion
    aeson >= 2.0 && < 2.2,
    base >= 4.6 && < 4.18,
```

(Support for GHC 9.4 and stackage nightly that supports 9.4, which uses aeson-2.1.2.1)

---

Done